app/controllers/conferences_controller.rb

```diff
@@ -12,16 +15,46 @@ def show
     @conference = Conference.find(params[:id])
   end
 
+  def create
+    @conference = Conference.new(conference_params)
+    generate_gid(@conference)
+    @team = Team.find(params[:team_id])
+
+    respond_to do |format|
+      if @conference.save
+        format.html { redirect_to params[:redirect_to].presence || edit_team_path(@team), notice: 'Conference was successfully created.' }
+        format.json { render action: :edit, status: :created, location: @team }
+      else
+        format.html { render action: :new }
+        format.json { render json: @conference.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
   def redirect
     redirect_to orga_conferences_path
   end
 
   private
+
+  def generate_gid(conference)
+    conference.gid = Time.now.getutc
+  end
  
   def conferences
     Conference.ordered(sort_params).in_current_season
   end
 
+  def conference_params
+    params.require(:conference).permit(
+      :name, :twitter, :starts_on, :ends_on, :notes, :country, :region, :location, :city, :url, :season_id
```

Comment:
If the assignment of the season is handled in the controller, the `season_id` can be removed from this list.